internal/pkg/admission/policy-server-ca-secret.go

```diff
@@ -97,27 +128,59 @@ func extractCaFromSecret(caSecret *corev1.Secret) (*admissionregistration.CA, er
 	return &admissionregistration.CA{CaCert: caCert, CaPrivateKey: caPrivateKey}, nil
 }
 
-func (r *Reconciler) fetchOrInitializePolicyServerCARootSecret(ctx context.Context, generateCA generateCAFunc, pemEncodeCertificate pemEncodeCertificateFunc) (*corev1.Secret, error) {
-	policyServerSecret := corev1.Secret{}
+// FetchOrInitializeRootCASecret checks if the root CA secret exists and
+// creates it if it does not.  It returns the secret containing the root CA, a
+// boolean flag to signal if the secret has been initialize now and an error in
```

Comment:
I like the bool `initialized` return. It paves the way for refreshing the CA out of the controller if users need.

Reply:
```suggestion
// boolean flag to signal if the secret has been initialized now and an error in
```